src/main/scala/com/ing/wbaa/rokku/sts/api/AdminApi.scala

```diff
@@ -62,4 +64,26 @@ trait AdminApi extends LazyLogging with Encryption {
     }
   }
 
+  def enableOrDisableAccount: Route =
```

Comment:
a little strange name for me :) - maybe 
```suggestion
  def setAccountStatus: Route =
```